src/components/buttons/DeleteButton.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react';
-
 import IconButton from 'material-ui/IconButton';
 import TrashCan from 'material-ui/svg-icons/action/delete';
 import ReactTooltip from 'react-tooltip';
+import { branch, renderNothing } from 'recompose';
```

Comment:
We're done working on this project next week, and I don't expect anyone to have to edit this component soon.